linebot/get_bot_info_test.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package linebot
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestGetBotInfo(t *testing.T) {
+	type want struct {
+		URLPath     string
+		RequestBody []byte
+		Response    *BotInfoResponse
+		Error       error
+	}
+	var testCases = []struct {
+		Label        string
+		ResponseCode int
+		Response     []byte
+		Want         want
+	}{
+		{
+			Label:        "Success",
+			ResponseCode: 200,
+			Response:     []byte(`{"userId":"u206d25c2ea6bd87c17655609a1c37cb8","basicId":"@012abcde","premiumId":"PremiumId","pictureUrl":"https://example.com/abcdefghijklmn","chatMode":"chat","markAsReadMode":"manual"}`),
+			Want: want{
+				URLPath:     APIEndpointGetBotInfo,
+				RequestBody: []byte(""),
+				Response: &BotInfoResponse{
+					UserID:         "u206d25c2ea6bd87c17655609a1c37cb8",
+					BasicID:        "@012abcde",
+					PremiumID:      "PremiumId",
+					PictureURL:     "https://example.com/abcdefghijklmn",
```

Comment:
Also for "displayName"

Refer https://developers.line.biz/en/reference/messaging-api/#get-bot-info-response